bitrise.yml

```diff
@@ -0,0 +1,192 @@
+format_version: 3
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  - BITRISE_STEP_ID: script-runner
+  - BITRISE_STEP_VERSION: "0.0.1"
+  - BITRISE_STEP_GIT_CLONE_URL: https://github.com/bitrise-steplib/steps-script-runner
+  - MY_STEPLIB_REPO_FORK_GIT_URL: $MY_STEPLIB_REPO_FORK_GIT_URL
+  - IS_DEBUG: "yes"
+
+workflows:
+  test:
+    before_run:
+    - go-tests
+    steps:
+    - change-workdir:
+        title: Switch working dir to test / _tmp dir
+        description: |-
+          To prevent step testing issues, like referencing relative
+          files with just './some-file' in the step's code, which would
+          work for testing the step from this directory directly
+          but would break if the step is included in another `bitrise.yml`.
+        run_if: true
+        inputs:
+        - path: ./_tmp
+        - is_create_path: true
+    - script:
+        title: Create test.sh file
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -e
+            echo "echo 'This is a shell script'" > test.sh
+        - is_debug: false
+    - script:
+        title: Create test.rb file
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -e
+            echo "puts \"This is a ruby script\"" > test.rb
+    - path::./:
+        title: Test Bash
+        description: |-
+          Test running a Bash script
+        run_if: true
+        inputs:
+        - file_path: "test.sh"
+        - working_dir: ./
+        - is_debug: $IS_DEBUG
+    - path::./:
+        title: Test Ruby
+        description: |-
+          Test running a Ruby script
+        inputs:
+        - file_path: "test.rb"
+        - working_dir: ./
+        - runner_bin: "ruby"
+        - is_debug: $IS_DEBUG
```

Comment:
Please add: 
- integration test for go script as well
- bash script with non 0 exit code